pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -30,6 +30,15 @@ import (
 
 var log = logf.Log.WithName("controller_baremetalhost")
 
+// FIXME(dhellmann): These values should probably come from
+// configuration settings and something that can tell the IP address
+// of the ironic server.
+const (
+	instanceImageSource   = "http://172.22.0.1/images/redhat-coreos-maipo-latest.qcow2"
+	instanceImageChecksum = "97830b21ed272a3d854615beb54cf004"
```

Comment:
Ironic has just added support for loading checksums from a URL. You may want to use it.

Reply:
Is that available in the version of ironic we'll be packaging for this?

Reply:
Ultimately these should come from a config somewhere.. I think the values are probably ok for now, except it might randomly break if the latest image gets updated

Reply:
I think we should be using a stein (or later) Ironic for this, but either way we shouldn't hard-code this checksum, because we exposed a file alongside the qcow2 that contains it:

https://github.com/metalkube/dev-scripts/blob/master/04_setup_ironic.sh#L76

So we should either read http://172.22.0.1/images/redhat-coreos-maipo-latest.qcow2.md5sum in the operator, or pass that URL to ironic so it can read it?

Reply:
We'll need IPA images from Stein (this feature is in IPA, not in Ironic itself). It's a bit tricky to figure out whether the RDO images have been updated already, but they certainly will have by Stein final.

Reply:
I agree we should update the images. I will open a separate ticket to track that.